applications/composer/deploy/values.yaml

```diff
@@ -44,3 +44,8 @@ harness:
     failureThreshold: 3
   envmap:
     PRODUCTION: True
+    SOCIAL_AUTH_ORCID_KEY: APP-GRRGRZ5EZQLQ6WZT
+    CSRF_TRUSTED_ORIGINS: "https://*.metacell.us/"
```

Comment:
this needs to be fixed for prod deployment since it is not hosted on metacell.us